File: PHP/uberPool.php
<?php
require_once('car.php');
class UberPool extends Car {
    //Atributos propios
    public $brand;
    public $model;

    //constructor
    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}
?>

File: C#/UberPool.cs
namespace UberPoo
{

    class UberPool : Car
    {
        public string brand;
        public string model;

        public UberPool(string license, Account driver, string brand, string model) : base(license, driver)
        {
            this.brand = brand;
            this.model = model;
        }
    }
}